ftw/contacts/profiles/simplelayout/types/ftw.contacts.Member.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0"?>
+<object name="ftw.contacts.MemberBlock"
+        meta_type="Dexterity FTI"
+        xmlns:i18n="http://xml.zope.org/namespaces/i18n"
+        i18n:domain="ftw.contacts" >
+
+    <!-- add permission -->
+    <property name="add_permission">ftw.contacts.AddMemberBlock</property>
+
```

Comment:
This types.xml overrides the ftw.contacts.Member type. This profile is only active if ftw.simplelayout is installed.

We don't want two different contenttypes, a Member and a MemberBlock. We just want one. The problem is, that the memberblock needs a different permission. And thats what i'm doing here...